libs/wingc/src/type_check.rs

```diff
@@ -764,7 +764,7 @@ impl<'a> TypeChecker<'a> {
 			vec![WINGSDK_STD_MODULE.to_string()],
 			&Symbol {
 				name: WINGSDK_STD_MODULE.to_string(),
-				span: WingSpan::global(),
+				span: Default::default(),
 			},
```

Comment:
(optional) For these cases we could consider adding a helper method perhaps:
```rust
Symbol::global(WINGSDK_STD_MODULE)
```

implemented as something like
```rust
fn global<S: Into<String>>(name: S) -> Self {
	Self { name: name.into(), span: Default::default() }
}
```

Reply:
Noice, yeah this looks good I went ahead and added it